src/index.js

```diff
@@ -294,7 +294,7 @@ class Frecency {
         if (selection) {
           result._frecencyScore = this._calculateScore(selection.selectedAt,
             selection.timesSelected, now);
-          return;
+          if (result._frecencyScore > 0) return;
```

Comment:
Why that?

Reply:
A 0 score means entry is too old (> 14 days).

in this case, we want to continue to fallback on other score computation (i.e. [line 314 (sub-query matching)](https://github.com/getstation/frecency/pull/1/files#diff-1fdf421c05c1140f6d71444ea2b27638R314) and [line 324 (recent selections matching)](https://github.com/getstation/frecency/pull/1/files#diff-1fdf421c05c1140f6d71444ea2b27638R324))

Reply:
Wouldn't it be better if we had a way to pipe all those algorithm through the score-engine, instead of letting the frecency handle it all ? What steps would be required to do so ?

Reply:
This is a reason why I think we have to rewrite our own frecency engine if we want to do that.

Reply:
@alexstrat I just write 2 tests to show the cases: 

- https://github.com/getstation/frecency/commit/51cdf5c2efcdfb7946c33543093b5a9b8ca0756f#diff-ccf0df6c26159476448a2b3b194b2a95R368
- https://github.com/getstation/frecency/commit/51cdf5c2efcdfb7946c33543093b5a9b8ca0756f#diff-ccf0df6c26159476448a2b3b194b2a95R407

I will go for a PR soon.